core/controllers/question_editor.py

```diff
@@ -56,8 +63,9 @@ def post(self, skill_id):
         question_services.add_question(self.user_id, question)
         # TODO(vinitamurthi): Replace DEFAULT_SKILL_DIFFICULTY
         # with a value passed from the frontend.
-        question_services.create_new_question_skill_link(
-            question.id, skill_id, constants.DEFAULT_SKILL_DIFFICULTY)
+        for skill_id in skill_ids:
+            question_services.create_new_question_skill_link(
```

Comment:
Update your branch,  I think this is all the old work of question_editor.

Reply:
Done.